src/components/FeatureViews/AdminPortal/AdminDashboard.js

```diff
@@ -171,6 +173,47 @@ const AdminDashboard = (props) => {
     { label: "Access Control" },
   ];
   
+  const QUERIES = {
+    "unpaidsessions": gql`query MyQuery {
+      unpaidSessions {
+        paymentList {
+          parent {
+            user {
+              firstName
+              lastName
+            }
+          }
+        }
+        course {
+          startTime
+          endTime
+          hourlyTuition
+        }
+        lastPaidSessionDatetime
+      }
+    }
+    `,
+  }
+
+// console.log(UNPAID_SESSIONS)
+
+ const unpaidSessionsData = useQuery(QUERIES["unpaidsessions"])
```

Comment:
separate this out to `const {data, loading, error}` remember in my demo how I handled the loading logic here. `loading` is a boolean, so if it's true then display a loading component.